stable/jenkins/templates/config.yaml

```diff
@@ -117,7 +117,7 @@ data:
     echo "false" > /usr/share/jenkins/ref/secrets/slave-to-master-security-kill-switch;
     cp -n /var/jenkins_config/config.xml /var/jenkins_home;
 {{- if .Values.Master.InstallPlugins }}
-    cp -n /var/jenkins_config/plugins.txt /var/jenkins_home;
+    cp /var/jenkins_config/plugins.txt /var/jenkins_home;
```

Comment:
I am not sure if this is the best approach.

It might be as good/better to remove this line and change 121 to 

```
/usr/local/bin/install-plugins.sh `echo $(cat /var/jenkins_config/plugins.txt)`;
```

Was there a reason for plugins.txt being copied into /var/jenkins_home/ anyway?